identity-enabler/deviceId-mobile-app/src/App.svelte

```diff
@@ -32,7 +32,7 @@
      * @param decodedText The content supplied by DataWedge (Zebra Scanner)
      */
     async function onScan(decodedText: string) {
-        if (navigator.onLine === false) {
+        if (!navigator.onLine) {
```

Comment:
Apparently, `onLine` cannot be undefined and so ES Lint sees the `false` check as unnecessary.

Reply:
better if we disable eslint in this line